opencog/atomspace/SimpleTruthValue.cc

```diff
@@ -74,16 +75,22 @@ confidence_t SimpleTruthValue::getConfidence() const
 // This is the merge formula appropriate for PLN.
 TruthValuePtr SimpleTruthValue::merge(TruthValuePtr other,TVMergeStyle ms/*=DEFAULT*/) const
 {
-    if (other->getType() != SIMPLE_TRUTH_VALUE) {
-        throw RuntimeException(TRACE_INFO,
-           "Don't know how to merge %s into a SimpleTruthValue",
-           typeid(*other).name());
-    }
-
-    if (other->getConfidence() > getConfidence()) {
-        return other->clone();
-    }
-    return clone();
+	switch(ms){
+	case DEFAULT:
+	{
+		if (other->getType() != SIMPLE_TRUTH_VALUE)
+		        throw RuntimeException(TRACE_INFO,
+		           "Don't know how to merge %s into a SimpleTruthValue using the default style",
+		           typeid(*other).name());
+		auto count2 = other->getCount();
```

Comment:
It would be nice to quote chapter and verse of the text which defines this formula.  I assume it is in some book or pdf somewhere or maybe it is discussed in a wiki page.